File: remote/src/main/java/com/kelsos/mbrc/ui/views/DeviceManagerView.java
package com.kelsos.mbrc.ui.views;

import com.kelsos.mbrc.events.ui.DiscoveryStopped;
import com.kelsos.mbrc.events.ui.NotifyUser;

public interface DeviceManagerView {
  void showDiscoveryResult(@DiscoveryStopped.Status long reason);

  void dismissLoadingDialog();

  void showNotification(NotifyUser event);
}

File: remote/src/main/kotlin/com/kelsos/mbrc/ui/views/DeviceManagerView.kt
package com.kelsos.mbrc.ui.views

import com.kelsos.mbrc.domain.DeviceSettings
import com.kelsos.mbrc.events.ui.DiscoveryStopped
import com.kelsos.mbrc.events.ui.NotifyUser

interface DeviceManagerView {
  fun showDiscoveryResult(@DiscoveryStopped.Status reason: Long)

  fun dismissLoadingDialog()

  fun showNotification(event: NotifyUser)

  fun updateDevices(list: List<DeviceSettings>)
}
